facebook/OpenBIC
# Artemis: Enable ADC deglitch

# Description
Set ADC deglitch.

# Motivation
Filter abnormal ADC value.

# Test Plan
1. Build and test pass on Artemis system. pass

2. Check MC BIC sensor reading root@bmc-oob:~# sensor-util mc -t | grep -E "MC_P3V3_AUX_VOL_V|MC_P1V2_AUX_VOL_V|MC_P3V3_VOL_V"
MC_P3V3_AUX_VOL_V            (0x5) :    3.34 Volts | (ok) | UCR: 3.53 | UNC: NA | UNR: NA | LCR: 3.07 | LNC: NA | LNR: NA
MC_P1V2_AUX_VOL_V            (0x6) :    1.21 Volts | (ok) | UCR: 1.28 | UNC: NA | UNR: NA | LCR: 1.12 | LNC: NA | LNR: NA
MC_P3V3_VOL_V                (0x7) :    3.34 Volts | (ok) | UCR: 3.53 | UNC: NA | UNR: NA | LCR: 3.07 | LNC: NA | LNR: NA

3. Check CB BIC sensor reading root@bmc-oob:~# sensor-util cb -t | grep -E "CB_P3V3_AUX_V|CB_P1V2_AUX_V|CB_P1V8_PEX_V|CB_P1V8_1_VDD_V|CB_P1V8_2_VDD_V|CB_P1V25_1_VDD_V|CB_P1V25_2_VDD_V|CB_P5V"
CB_P5V_AUX_V                 (0x10) :    5.13 Volts | (ok) | UCR: 5.25 | UNC: NA | UNR: NA | LCR: 4.75 | LNC: NA | LNR: NA
CB_P3V3_AUX_V                (0x11) :    3.34 Volts | (ok) | UCR: 3.53 | UNC: NA | UNR: NA | LCR: 3.07 | LNC: NA | LNR: NA
CB_P1V2_AUX_V                (0x12) :    1.21 Volts | (ok) | UCR: 1.28 | UNC: NA | UNR: NA | LCR: 1.12 | LNC: NA | LNR: NA
CB_P1V8_PEX_V                (0x13) :    1.81 Volts | (ok) | UCR: 1.89 | UNC: NA | UNR: NA | LCR: 1.71 | LNC: NA | LNR: NA
CB_P1V8_1_VDD_V              (0x1E) :    1.84 Volts | (ok) | UCR: 1.89 | UNC: NA | UNR: NA | LCR: 1.71 | LNC: NA | LNR: NA
CB_P1V8_2_VDD_V              (0x1F) :    1.83 Volts | (ok) | UCR: 1.89 | UNC: NA | UNR: NA | LCR: 1.71 | LNC: NA | LNR: NA
CB_P1V25_1_VDD_V             (0x20) :    1.27 Volts | (ok) | UCR: 1.31 | UNC: NA | UNR: NA | LCR: 1.19 | LNC: NA | LNR: NA
CB_P1V25_2_VDD_V             (0x21) :    1.28 Volts | (ok) | UCR: 1.31 | UNC: NA | UNR: NA | LCR: 1.19 | LNC: NA | LNR: NA
